playground/ExternalServices/ExternalServices.AppHost/aspire-manifest.json

```diff
@@ -41,29 +41,4 @@
     },
     "frontend": {
       "type": "project.v0",
-      "path": "../WebFrontEnd/WebFrontEnd.csproj",
-      "env": {
-        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES": "true",
-        "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
-        "OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY": "in_memory",
-        "ASPNETCORE_FORWARDEDHEADERS_ENABLED": "true",
-        "HTTP_PORTS": "{frontend.bindings.http.targetPort}",
-        "services__nuget__https__0": "https://api.nuget.org/",
-        "EXTERNAL_SERVICE_URL": "{external-service-url.value}",
-        "services__gateway__http__0": "{gateway.bindings.http.url}"
-      },
-      "bindings": {
-        "http": {
-          "scheme": "http",
-          "protocol": "tcp",
-          "transport": "http"
-        },
-        "https": {
-          "scheme": "https",
-          "protocol": "tcp",
-          "transport": "http"
-        }
-      }
-    }
-  }
-}
\ No newline at end of file
+      "path": "../WebFrontEnd/WebFrontEnd.csproj"
```

Comment:
I've opened [azd init fails when External Service Parameter has no default value and configuration key is missing (dotnet/aspire#10789)](https://github.com/dotnet/aspire/issues/10789) for this. Generating a manifest for this app is broken and throwing an exception.